pandas/core/frame.py

```diff
@@ -559,14 +559,14 @@ def _is_homogeneous_type(self) -> bool:
     # ----------------------------------------------------------------------
     # Rendering Methods
 
-    def _repr_fits_vertical_(self):
+    def _repr_fits_vertical_(self) -> bool:
```

Comment:
i wonder whether this method and `_repr_fits_horizontal_` should belong in this module.

Reply:
Yea I think open to moving if it makes sense elsewhere (separate PR)

Reply:
> separate PR

agreed, no action required here